Artifacts/MsSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql

```diff
@@ -0,0 +1,33 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE @applicationId INT;
+DECLARE @systemDescriptorsResourceClaimId INT;
+DECLARE @relationshipBasedDataResourceClaimId INT;
+
+SELECT @applicationId = (SELECT applicationid FROM  dbo.Applications  WHERE  ApplicationName  = 'Ed-Fi ODS API');
+SELECT @systemDescriptorsResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'systemDescriptors' AND Application_ApplicationId = @applicationId);
+SELECT @relationshipBasedDataResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'relationshipBasedData' AND Application_ApplicationId = @applicationId);
+
+/* new descriptors */
+IF (NOT EXISTS (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'AncestryEthnicOriginDescriptor' AND Application_ApplicationId = @applicationId))
+BEGIN
+	INSERT dbo.ResourceClaims (DisplayName, ResourceName, ClaimName, ParentResourceClaimId, Application_ApplicationId)
+	VALUES (N'AncestryEthnicOriginDescriptor', N'AncestryEthnicOriginDescriptor', N'http://ed-fi.org/ods/identity/claims/AncestryEthnicOriginDescriptor', @systemDescriptorsResourceClaimId, @applicationId);
+END
+
+-- Try insert StudentDisciplineIncidentBehaviorAssociation
+IF (NOT EXISTS (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'StudentDisciplineIncidentBehaviorAssociation' AND Application_ApplicationId = @applicationId))
+BEGIN
+INSERT [dbo].[ResourceClaims] ([DisplayName], [ResourceName], [ClaimName], [ParentResourceClaimId], [Application_ApplicationId])
```

Comment:
See comment on Postgres script below regarding proper formatting and indenting.

Reply:
yes ,I fixed it . Thanks